helpers/DeployHelper.ts

```diff
@@ -584,20 +592,22 @@ export class DeployHelper {
         break;
       case "v3":
         verification = contractsInfo.GROTH16_VERIFIER_V3.verificationOpts;
+      case "authV2":
+        verification = contractsInfo.GROTH16_VERIFIER_AUTH_V2.verificationOpts;
         break;
     }
     return verification;
   }
 
-  getValidatorVerification(validatorType: "mtpV2" | "sigV2" | "v3"): {
+  getValidatorVerification(verifierType: VerifierType): {
```

Comment:
Using the same types for both Groth16Verifiers and Validators is not the best way of organisation. It's not the same. In addition, in the future we may have more than one Groth16Verifier per Validator